scripts/groovy/gather_jenkins_facts.groovy

```diff
@@ -52,10 +52,14 @@ Map getPluginInfo(List<PluginWrapper> plugins) {
   Map ret = [:]
   for (PluginWrapper plugin in plugins) {
     UpdateSite.Plugin updateInfo = plugin.getUpdateInfo()
+    if (updateInfo != null) {
+        isCompatibleWithInstalledVersion = updateInfo.isCompatibleWithInstalledVersion()
```

Comment:
I think we should declare isCompatibleWithInstalledVersion properly as boolean and initialize it with a default value of false for the case when `updateInfo` is `null`

```
Boolean isCompatibleWithInstalledVersion = false
if (updateInfo != null) {
  isCompatibleWithInstalledVersion = updateInfo.isCompatibleWithInstalledVersion()
}
```

Reply:
added 
Boolean isCompatibleWithInstalledVersion = false

Reply:
thanks!